guides/common/modules/proc_performing-a-full-backup.adoc

```diff
@@ -6,8 +6,14 @@
 There are two main methods of backing up {ProjectServer}:
 
 * Offline backup
++
+All {Project} services are shut down during an offline backup.
+
 * Online backup
 +
+With online backups, only services that affect the consistency of the backup are shut down while the backup process is running.
```

Comment:
This is rather vague from a user's perspective. Do we have a place where we go into more detail about what is possible and what is not possible during online backup capability wise?

Reply:
We do not

Reply:
We can create a concept module for this with more detailed description.

Reply:
Let's either do that as part of this or create a tracker to tackle it as a follow up.